### ISSUE TITLE
Server crash when Lobster loaded

### ISSUE THREAD

**Issue by treymur** [tomalbrc/toms-mobs#8]
Spawned in a lobster on land, which crashed the game and crashes the server every time the lobster is loaded:
```
[10:31:50] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking entity
        at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1073) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:953) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:713) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:292) ~[server-intermediary.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.IllegalArgumentException: Can't find attribute minecraft:tempt_range
        at knot/net.minecraft.class_5132.method_26865(class_5132.java:21) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_5132.method_26862(class_5132.java:27) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_5131.method_26852(class_5131.java:68) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1309.method_45325(class_1309.java:2044) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1391.method_6264(class_1391.java:47) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_4135.method_6264(class_4135.java:22) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1355.method_6275(class_1355.java:87) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1308.method_6023(class_1308.java:798) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1309.method_6007(class_1309.java:2859) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1308.method_6007(class_1308.java:571) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1296.method_6007(class_1296.java:129) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1429.method_6007(class_1429.java:67) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1309.method_5773(class_1309.java:2595) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1308.method_5773(class_1308.java:371) ~[server-intermediary.jar:?]
        at knot/de.tomalbrc.toms_mobs.entities.passive.Lobster.method_5773(Lobster.java:77) ~[toms_mobs-2.1.2+1.21.2.jar:?]
        at knot/net.minecraft.class_3218.method_18762(class_3218.java:761) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_1937.method_18472(class_1937.java:501) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3218.method_31420(class_3218.java:406) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_5574.method_31791(class_5574.java:54) ~[server-intermediary.jar:?]
        at knot/net.minecraft.class_3218.method_18765(class_3218.java:376) ~[server-intermediary.jar:?]
        at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1069) ~[server-intermediary.jar:?]
        ... 4 more
```

**Comment by treymur** [comment]
just saw this is fixed with 2.1.3, is it possible to port this to be able to use with older version such as 1.21.3?

**Comment by tomalbrc** [comment]
Yep, 2.1.3 is available for 1.21.1 and 1.21.2-1.21.3 on modrinth
